07-performance.Rmd

```diff
@@ -71,11 +84,13 @@ operation.
 
 There are currently three sorting algorithms, `c("shell", "quick", "radix")` that can
 be specified in the `sort` function; with `radix` being a new addition to R 3.3.
-Typically the `radix` (the non-default option) is optimal for most situations.
-<!-- __XXX__Colin__ by how much? -->
+Typically the `radix` (the non-default option) is optimal for most situations (it is around 
+20% faster when sorting a large vector of doubles).
```

Comment:
Suggest replacing `optimal` with `the most computationally efficient option` here.

Why? There may be an improvement in the future that means it's not optimal, just the best of the current crop.